index.js

```diff
@@ -34,3 +34,34 @@ exports.getFan = function(number) {
 exports.getNumberOfFans = function() {
     return binding.GetFans();
 };
+
+exports.getTemp = function(tempKey, units) {
+    if (!tempKey)
+        throw new Error('Must pass a valid temperature key to read');
+
+    var unitType = 0;
+
+    if(units) {
+        // We unify this for checking
+        units = units.toUpperCase();
+
+        switch(units)
+        {
+        case 'FAHRENHEIT':
+            unitType = 1;
+            break;
+
+        case 'KELVIN':
+            unitType = 2;
+            break;
+
+        case 'CELSIUS':
+        default:
+            unitType = 0;
+        }
+    }
+
+    var info = binding.GetTemp(tempKey, unitType);
+
+    return info || 0;
```

Comment:
Just wondering, why is `|| 0` used here?

Reply:
Oops, that was left over from some of my testing/debugging. It shouldn't actually be required.